resqpy/well.py

```diff
@@ -3007,6 +3012,12 @@ def xyz_marker(self, active_only = True):
       xyz = self.trajectory.xyz_for_md(md)
       return xyz, rqet.uuid_for_part_root(self.trajectory.crs_root)
 
+   def create_feature_and_interpretation(self):
```

Comment:
Could have a docstring if it's a public method (or else, could give an underscore prefix to the name to make this a private method):

```suggestion
   def create_feature_and_interpretation(self):
      """Instantiate new empty WellboreFeature and WellboreInterpretation objects
      
      Uses the Blocked well citation title as the well name
      """
```